src/healthcareapis/HISTORY.rst

```diff
@@ -3,6 +3,24 @@
 Release History
 ===============
 
+0.4.0
+++++++
+
+* Bump version to 2021-11-01 stable.
```

Comment:
Better to remove this note since CLI customers don't care the underlying API version.